e2e/scripts/st_multiselect.py

```diff
@@ -15,6 +15,13 @@
 from typing import Any, List
 
 import streamlit as st
+from enum import Enum
+
+
+class Colors(Enum):
```

Comment:
Would be good to also test an enum whose values are strings, since that's the other common way people write them